src/app/dim-ui/PressTip.m.scss

```diff
@@ -38,34 +32,35 @@ $horizontal-padding: 11px;
     content: '';
     display: block;
     position: absolute;
-    background-color: var(--tooltip-ribbon-color);
-    height: $ribbon-width;
-    top: -$ribbon-width;
+    background-color: var(--theme-tooltip-border-color);
+    height: $theme-tooltip-corner-radius;
+    top: calc(-1 * $theme-tooltip-corner-radius);
     left: -1px;
     right: -1px;
-    border-radius: $ribbon-width $ribbon-width 0 0;
+    border-radius: $theme-tooltip-corner-radius $theme-tooltip-corner-radius 0 0;
   }
 
   &.wideTooltip {
     max-width: 100vw;
   }
 
   hr {
-    border-color: dim-rgb-values-to-rgba(var(--tooltip-border-color-rgb));
+    border-color: var(--theme-tooltip-border-color);
   }
 
   p {
     margin-bottom: 0;
   }
 
   .header {
-    background-color: #0a0a0f;
+    background-color: var(--theme-fill-secondary);
```

Comment:
This variable appears to be missing:

![dim-tooltip-missing-header](https://github.com/DestinyItemManager/DIM/assets/17512262/34ad05b9-adff-4306-bc96-6e557b6f701e)

Reply:
Added a var for this